app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx

```diff
@@ -105,7 +105,10 @@ export class BtsPropertiesForIssueForm extends Component {
 
   constructor(props) {
     super(props);
-    this.defaultOptionValueKey = this.props.pluginName === JIRA ? VALUE_NAME_KEY : VALUE_ID_KEY;
+    this.defaultOptionValueKey =
+      this.props.pluginName === JIRA || this.props.pluginName === AZURE
+        ? VALUE_NAME_KEY
+        : VALUE_ID_KEY;
```

Comment:
For now, we try to avoid mentioning third parties in our core app. So it will be refactored soon.
In your case, you can use `getDefaultOptionValueKey` here with a default `VALUE_NAME_KEY` appropriate for your Azure integration.

Reply:
Did it.